build.sbt

```diff
@@ -678,7 +678,12 @@ val generateRustParserLib =
     allLibs.isEmpty ||
     (`syntax-rust-definition` / generateRustParserLib).inputFileChanges.hasChanges
   ) {
-    Seq("cargo", "build", "-p", "enso-parser-jni") !
+    val os = System.getProperty("os.name")
+    if (os.startsWith("Mac")) {
+        Seq("cargo", "build", "-p", "enso-parser-jni", "--target", "x86_64-apple-darwin") !
```

Comment:
Is this the only architecture that we support for Macs?
If not, shouldn't we rather check the arch directly instead of indirectly by checking the OS?